provider/resource_keycloak_realm.go

```diff
@@ -76,17 +76,13 @@ func resourceKeycloakRealm() *schema.Resource {
 			//Smtp server
 
 			"smtp_server": {
-				Type:     schema.TypeSet,
+				Type:     schema.TypeList,
```

Comment:
This won't change how this attribute is used, but it's much easier to use `TypeList` for one-off block attributes.

Reply:
ok